pkg/volume/glusterfs/glusterfs.go

```diff
@@ -309,7 +309,7 @@ func (b *glusterfsMounter) setUpAtInternal(dir string) error {
 
 	p := path.Join(b.glusterfs.plugin.host.GetPluginDir(glusterfsPluginName), b.glusterfs.volName)
 	if err := os.MkdirAll(p, 0750); err != nil {
-		return fmt.Errorf("glusterfs: mkdir failed: %v", err)
```

Comment:
Why is the directory in square brackets?

`glusterfs: mkdir [/var/lib/kubelet/plugins/kubernetes.io/glusterfs/xyz] failed: Permission denied`.

What about `Error creating directory /var/lib/kubelet/plugins/kubernetes.io/glusterfs/xyz: Permission denied`?

I don't like `glusterfs` prefix everywhere too, but that's probably for another PR.